tests/test_ccc.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from tests.utils import file_response
+from documenters_aggregator.spiders.ccc import CccSpider
+
+#def test_tests():
+#    print('Please write some tests for this spider or at least disable this one.')
+#    assert False
+
+
+"""
+Uncomment below
+"""
+
+test_response = file_response('files/ccc.html')
```

Comment:
What file_response does is take a file (ccc.html) that we've saved locally and create a Response object from it, so that we can test out the spider's parse method without needing an Internet connection. (The website might change, but we don't want that to break our tests.) Since your parse method goes to a different website for each event, we can just test the _parse_event_page function on a single event's site. You can do something like this:
- Save one of the event pages in tests/files/  (for example, save this site: http://www.ccc.edu/events/Pages/Committee-on-Finance-and-Administrative-Services-Sept.-2017.aspx as tests/files/ccc_event.html)
- Code: 
```python
test_response = file_response('files/ccc_event.html')
spider = CccSpider()
item = spider._parse_event_page(test_response)
# write your tests as usual using item instead of parsed_items[0]
```